azure/terraform/Redis.rego

```diff
@@ -19,12 +19,15 @@ azure_issue ["enableSslPort"] {
 }
 
 enableSslPort {
+    lower(input.resources[_].type) == "azurerm_redis_cache"
     azure_attribute_absence["enableSslPort"]
+    not azure_issue["enableSslPort"]
 }
 
```

Comment:
@rezoan there is no need for ```lower(input.resources[_].type) == "azurerm_redis_cache"```
because ```azure_attribute_absence``` will check that
we used that when we want to check ```not azure_attribute_absence```
like this:
```
enableSslPort {
    lower(input.resources[_].type) == "azurerm_redis_cache"
    not azure_issue["servenableSslPorterRole"]
    not azure_attribute_absence["enableSslPort"]
}
```

Reply:
there was a situation where we need to handle 2 types of resource (one parent and one child). for that type of case we need to check the parent type explicitly in calling rule as azure_attribute_absence and azure_issue will check the child type.
But for single resource type you are correct.
Checking the type for single resource would nor hurt but i will remove those checking for single type resoruce in next PR.